### DISCUSSION TITLE
Error when saving beatmap (with ctrl + s) while exporting it as olz or osz

### DISCUSSION THREAD

**Discussion by hwabis** [ppy/osu#25425] [Q&A] [answered]
[runtime.log](https://github.com/ppy/osu/files/13344191/runtime.log)

https://github.com/ppy/osu/assets/62194990/42eed7e9-24df-4dfd-be63-064f2e4c4a7a

**Comment by peppy** [comment, marked as answer]
Thanks for the report. Tracking at https://github.com/ppy/osu/issues/25426.
